lib/reek/context_builder.rb

```diff
@@ -102,7 +102,7 @@ def process_send(exp)
       if exp.attribute_writer?
         exp.args.each do |arg|
           next unless arg.type == :sym
-          new_context(Context::AttributeContext, arg, exp)
+          append_new_context(Context::AttributeContext, arg, exp)
```

Comment:
The new name makes it actually clear that we calling this for the side effect, and not for the return value.